README.md

```diff
@@ -12,10 +12,12 @@ _Inspired by [Awesome DynamoDB](https://github.com/alexdebrie/awesome-dynamodb)_
 
 ## Table of Contents
 
-- [Videos](#videos)
```

Comment:
Cool all looking good to me, thanks for fixing typos too.

Any reason the TOC has changed? Can't see why it would need too? Maybe we can revert the table on contents changes?

Reply:
Sorry, let me fix that! Seems my ToC extension messed it up ;)